ossm-threescale-authentication.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-authentication_{context}"]
+= Authenticating requests
+This release supports the following authentication methods:
```

Comment:
This line isn't rendered in the preview.

Reply:
This will be addressed in a different PR.